content/en/docs/reference/score-spec-reference.md

```diff
@@ -112,7 +112,7 @@ In general, `resource-type` has no meaning for Score, but it can affect how the
 | ------------- | ------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------- |
 | `environment` | Translates to the environment variables references. For example: `${PROPERTY-NAME}`.                                            | Translates to the application values references. For example: `${values.property-name}`.                                        |
 | `volume`      | Translates into a reference to the external volume. This reference is usually used in a container’s volume mount specification. | Translates into a reference to the external volume. This reference is usually used in a container’s volume mount specification. |
-| `workload`    | N/A                                                                                                                             | Translates to the module properties references. For example: `${modules.workload-name.property-name}`.                          |
```

Comment:
We're keeping `workload` as a reserved resource type. For score-compose and score-humanitec we throw the following error "Warning: workload is a reserved resource type. Its usage may lead to compatibility issues with future releases of score-humanitec"